modules/openiddict/src/Volo.Abp.OpenIddict.Domain.Shared/Volo.Abp.OpenIddict.Domain.Shared.csproj

```diff
@@ -23,4 +23,8 @@
     <Content Remove="Volo\Abp\OpenIddict\Localization\OpenIddict\*.json" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Remove="Volo\Abp\OpenIddict\Localization\OpenIddict\pt-BR.json" />
```

Comment:
I guess this was added automatically by IDE, can you remove it?

Reply:
Done @berkansasmaz, thank you!